pkg/helm/values.go

```diff
@@ -0,0 +1,21 @@
+package helm
+
+import (
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type HelmValues map[string]interface{}
```

Comment:
Should we have methods for other types (int, struct, etc) as well?

Reply:
Not yet. Would be dead code.